src/NodeManipulator/ClassInsertManipulator.php

```diff
@@ -154,20 +154,9 @@ private function hasClassConstant(Class_ $class, string $constantName): bool
         return false;
     }
 
-    /**
-     * Waits on https://github.com/nikic/PHP-Parser/pull/646
-     */
     private function hasClassProperty(Class_ $class, string $name): bool
     {
-        foreach ($class->getProperties() as $property) {
-            if (! $this->nodeNameResolver->isName($property, $name)) {
-                continue;
-            }
-
-            return true;
-        }
-
-        return false;
+        return $class->getProperty($name) !== null;
```

Comment:
probably better use : 

```suggestion
        return $class->getProperty($name) instanceof Property;
```